docs/book/src/development/development.md

```diff
@@ -138,6 +138,17 @@ You must configure systemd and iptables as described in https://kind.sigs.k8s.io
 
 You must install the `podman-docker` package to emulate the docker cli tool. However, this is not sufficient on its own as described below.
 
+### Configure slirp4netns to use a different CIDR
+
+By default we configure the controller and worker on 10.0.2.0/24. Rootless podman uses slirp4netns for networking, and unfortunately by default it uses the same CIDR internally. This prevents the management cluster from being able to route to the CI devstack.
+
+To fix this, add the following to `~/.config/containers/containers.conf`:
```

Comment:
just curious and education purpose :) , 
this is for containerd or cri-o or docker? 
can you point the issue about why we are suffering this to give more background on rootless podman 
etc so anyone hit the issue will be easier to understand the reason behind that?

Reply:
The issue is specific to a management kind cluster running in rootless podman, which we use when running the E2E tests. The kind cluster uses 10.0.2.0/24 internally, so it can't route to the controller and worker.

As suggested by @hidekazuna I changed the IPs of the controller and worker instead so this workaround is no longer necessary.